lib/ebooks/twitter.rb

```diff
@@ -18,9 +18,34 @@ def twitter_client
     end
 
     def tweet(tweet_text)
-      tweet_text = tweet_text.gsub('@', '')[0..139]
+      tweet_text = truncate tweet_text.gsub('@', '')
```

Comment:
I have a feeling this won't find `Ebooks.truncate` by itself. Am I incorrect? If so, why?